pyproject.toml

```diff
@@ -35,7 +35,7 @@ python = ">=3.9,<3.13"
 deepdiff = ">=5.5.0,!=6.0,!=6.1,<8"
 django-pivot = ">=1.9.0,<1.10.0" # The signature changed to return a non-queryset, do not upgrade without ensuring it returns a queryset
 matplotlib = "^3.3.2"
-nautobot-plugin-nornir = "^2.0.0"
+nautobot-plugin-nornir = "^2.2.1"
```

Comment:
Can you update this line as well? This was pointed out as an issue. 
nautobot = ">=2.4.2,<3.0.0"